Packages/com.unity.inputsystem/InputSystem/Devices/Gamepad.cs

```diff
@@ -146,7 +146,7 @@ public class Gamepad : InputDevice, IDualMotorRumble
         public ButtonControl startButton { get; private set; }
         public ButtonControl selectButton { get; private set; }
 
-        public DpadControl dpad { get; private set; }
+        public DpadControl dpad { get; protected set; }
```

Comment:
This is not a good change. The accessors sit on top of a contract established by a specific template. For Gamepad, the guarantee is that there is a DpadControl that can be found by looking for the name "dpad". AndroidGamepad breaks that contract.

I'll fix it up and bring it over to using a template constructor after the merge.